packages/discovery-provider/src/tasks/entity_manager/entities/track.py

```diff
@@ -657,33 +643,20 @@ def validate_update_access_conditions(params: ManageEntityParameters):
     existing_track = helpers.model_to_dictionary(
         params.existing_records["Track"][track_id]
     )
+    # scheduled release tracks are not restricted on how they can be updated
+    if existing_track.get("is_scheduled_release"):
+        return
```

Comment:
interesting... is it possible to have a scheduled-release track that has invalid conditions then?